core/pkg/server/handler.go

```diff
@@ -265,6 +266,9 @@ func (h *Handler) handleRecord(record *service.Record) {
 		h.handleUseArtifact(record)
 	case *service.Record_WandbConfigParameters:
 		h.handleWandbConfigParameters(record)
+	case *service.Record_ConfigFileParameter:
+		h.writeAndSendConfigFile(x.ConfigFileParameter)
```

Comment:
Is it appropriate for the copying to happen in handler? 

I doubt it will be common for this to be called more than once, and typically before a user has started calling `wandb.log`

Reply:
could you follow the handle<RecordName> convention plz :)

Reply:
ah yes missed this during a rename

Reply:
wait I'm actually not sure I understand what you meant

Reply:
`handlConfigFileParameter`